dmutils/api_stubs.py

```diff
@@ -213,3 +215,87 @@ def supplier(id=1234, contact_id=4321, other_company_registration_number=0, comp
         data['suppliers']['registrationCountry'] = 'country:NZ'
 
     return data
+
+
+def supplier_framework(
+    agreed_variations=True,
```

Comment:
It's probably not the end of the world with this as-is, but it would maybe be nice to have this able to inject actual/different/multiple variations. _Maybe_.

Reply:
I think I'll leave that as a task for if/when it comes up.